knpu/facebook.rst

```diff
@@ -234,17 +227,16 @@ right, and it can be done with the help of the SDK::
 When we try the process again, we get a valid-looking user id. So, what just
 happened?
 
-The ``getUser`` method does a whole lot more than it looks like. It actually
-looks for the ``code`` query parameter and makes the API request to get the 
+The `getUser` method does a whole lot more than it looks like. It actually
```

Comment:
Let's add `()` to all method names. If you edit a page and accidentally stumble on method or function without `()` - just add it! It's not much important, so don't look for this specifically ;)